config.py

```diff
@@ -0,0 +1,27 @@
+"""Flask config."""
```

Comment:
Oops, also I just noticed, remember the copyright header!

Reply:
Oh right, thanks for the catch!